ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -401,6 +388,14 @@ void OwInterface::managePanTilt (const string& opname,
 }
 
 ///////////////////////// Antenna/Camera Support ///////////////////////////////
+
+bool OwInterface::anglesEquivalent (double deg1, double deg2, double tolerance)
+{
+  return (fabs(deg1 - deg2) <= tolerance ||
+          fabs((deg1 - 360) - deg2) <= tolerance ||
+          fabs((deg1 + 360) - deg2) <= tolerance);
+}
```

Comment:
Would work better as the form discussed in the other PR.

Reply:
Agreed, done.